### PR TITLE
Filter out invalid CCs 

### PR DESCRIPTION
**Pull request by rmistry**
Context in b/324686394

### CODE REVIEW

**Comment by rmistry** [comment]
@jonathanmetzman PTAL

File: src/clusterfuzz/_internal/cron/cleanup.py [hunk 1]
```diff
@@ -1079,6 +1079,20 @@ def update_component_labels(policy, testcase, issue):
...
+  """Remove and log all entries with trailing comments.
+
+  Eg: Do not add "xyz@test.com #{LAST_RESORT_SUGGESTION}".
+  """
+  ret_list = []
+  for cc in ccs_list:
+    if len(cc.split()) == 1:
+      ret_list.append(cc)
+    else:
+      logs.log(f'google_issue_tracker: Filtering out CC "{cc}"')
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/cron/cleanup.py:1092]: Can you change this to `{cc!r}` from `"{cc}"`

**Review by jonathanmetzman — APPROVED** [review]

**Comment by rmistry** [comment]
Thanks! Please deploy whenever you get a chance

File: src/clusterfuzz/_internal/cron/cleanup.py (hunk 1 above)
Comment (rmistry) [reply src/clusterfuzz/_internal/cron/cleanup.py:1092]: Oops I completely missed this comment. Will piggyback this change into a future PR.

**Pull request merged**